### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by akpsgit**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+#### What happened:
+
+#### What you expected to happen:
+
+#### How to reproduce it (as minimally and precisely as possible):
+
+#### Anything else we need to know?:
+
+#### Environment:
```
Comment (FrimIdan) [inline .github/ISSUE_TEMPLATE/bug_report.md:22]: what about APIClarity version(we might need to integrate the version and commit-sha in the build phase to be able to extract it easily)? APIClarity logs?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- Kubernetes version (use `kubectl version`):
```
Comment (FrimIdan) [inline .github/ISSUE_TEMPLATE/bug_report.md:23]: `kubectl version --short`

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
```
Comment (FrimIdan) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: I see that Istio lately moved PR to a form (https://github.com/istio/istio/commit/7f810ef1b1c84e6e0d5442d60aaaf5d4c14e6d26)
maybe we should also do it? 
you can do validations for required fields and it looks better I think

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (akpsgit) [reply .github/ISSUE_TEMPLATE/bug_report.md:22]: Didn't put the version, as we don't have versions yet. Sure, adding the logs

**Review by FrimIdan — APPROVED** [review]

**Pull request merged**